src/composer.json

```diff
@@ -15,7 +15,8 @@
         "symfony/http-foundation": "2.6.*",
         "symfony/dependency-injection": "2.6.*",
         "twig/twig": "1.*",
-        "twig/extensions": "1.*"
+        "twig/extensions": "1.*",
+        "jumbojett/openid-connect-php": "0.1.*"
```

Comment:
If I understand this correctly, we already have Apache-2.0 in the Fossology project as a dependency
https://github.com/fossology/fossology/blob/master/NOTICES#L4068